kiwi/system/setup.py

```diff
@@ -115,7 +115,7 @@ def import_repositories_marked_as_imageinclude(self):
         imageinclude attribute should be permanently added to
         the image repository configuration
         """
-        repository_sections = self.xml_state.get_repository_sections()
+        repository_sections = self.xml_state.get_image_repository_sections()
```

Comment:
Reason for the name change is that I was confused what an image_repository is. With the background of the existing imageinclude and imageonly attributes it's clear but I think we could make it more explicit by a name change of the functions. What do you think ?